frontend/src/views/Game.tsx

```diff
@@ -1,13 +1,44 @@
 import Editor from '@monaco-editor/react';
 import React from 'react';
+import { Problem, getProblems } from '../api/problem';
+import styles from './styles.module.css';
 
-function GamePage() {
-  return (
-    <div>
-      Game page
-      <Editor height="90vh" language="javascript" />
-    </div>
-  );
+interface GameState {
+  problems?: Problem[];
+}
+
+interface GameProps {}
+
+class GamePage extends React.Component<GameProps, GameState> {
+  constructor(props: GameProps) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    getProblems().then((problems) => {
+      this.setState({ problems });
+    }).catch((error) => {
+      console.log(error);
+    });
+  }
+
+  render() {
+    const { problems } = this.state;
+    const firstProblem = problems?.[0];
+    return (
+      <div>
+        <div className={styles.header}><h3>Game page</h3></div>
+        <div className={styles.problemPanel}>
+          <h3>{ firstProblem?.name }</h3>
+          <p>{ firstProblem?.description}</p>
+        </div>
+        <div className={styles.editorPanel}>
+          <Editor height="90vh" language="javascript" />
+        </div>
```

Comment:
nit: Could you replace `<p>` with `Text` from `src/components/core/Text.tsx` and create a new component for `<h3>` along the lines of the other components there? That way these font sizes can be reused on other pages too.